.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
```

Comment:
```suggestion
      interval: weekly
```
Weekly is good enough for these.

Reply:
@ludeeus But why? If it can be daily, why not?

Reply:
It doesn't cost anyone resources if there is no update available.

Reply:
Because someone has to review these